db/re/item_db.conf

```diff
@@ -126639,10 +126639,10 @@ item_db: (
 	EquipLv: 95
 	Subtype: "W_MACE"
 	Script: <"
-		bonus bUnbreakableWeapon,0;
-		bonus3 bAutoSpell,NPC_CRITICALWOUND,5,10;
-		bonus bHit,getrefine();
-		bonus bCritical,getrefine();
+		bonus bUnbreakableWeapon, 0;
+		bonus3 bStateNoRecoverRace, RC_Player, 10000, 10000;
+		bonus bHit, getrefine() / 2;
```

Comment:
Is this bHit change intentional or accidental? There's no mention of it in the commit message

Reply:
oh i will separate this under another commit within the PR. this was from the sc file.